app/scripts/components/dashboardComponents/breadcrumbs.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react'
+const { object, string } = PropTypes;
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import { observer } from 'mobx-react';
+import mainStore from '../../stores/mainStore';
+import dashboardStore from '../../stores/dashboardStore';
+import { Color } from '../../theme/customTheme';
+import { Path } from '../../util/urlEnum';
+import FlatButton from 'material-ui/FlatButton';
+import Paper from 'material-ui/Paper';
+
+@observer
+class Breadcrumbs extends Component {
+    render() {
+        const { drawer, selectedItem } = dashboardStore;
+        const contentStyle = drawer.get('contentStyle')
+        const drawerDirection = drawer.get('open') ? '<' : '>'
+
+        return (
+            <div style={contentStyle}>
+                <Paper style={styles.breadCrumb}>
+                    <FlatButton
+                        label={drawerDirection}
+                        onClick={() => dashboardStore.toggleDrawer()}
+                        style={styles.drawerToggle}
+                    />
+                    <FlatButton
+                        label={drawer.get('toggleLable')}
+                        style={selectedItem ? styles.breadCrumb : styles.breadCrumbSelected}
+                        onClick={() => dashboardStore.toggleCollapseTree(this.props.router)}
+                    />
+                    {this.breadCrumb()}
+                </Paper> 
+          	</div>
+        );
+    }
+
+    pathFinder(kind) {
+        let kinds = {
```

Comment:
There's an enum for kinds in `util/urlEnum.js`. That way if an api change is made we can just change the kind string in one place.

Reply:
Changed it to a case switch so I could use Kind. I just pushed up those changes.